modelforge/tests/test_models.py

```diff
@@ -11,20 +10,35 @@
 )
 
 
-def test_BaseNNP():
-    nnp = BaseNNP()
-
-
 @pytest.mark.parametrize("model_class", MODELS_TO_TEST)
 @pytest.mark.parametrize("dataset", DATASETS)
 def test_forward_pass(model_class, dataset):
     for lightning in [True, False]:
         initialized_model = setup_simple_model(model_class, lightning)
-        inputs = return_single_batch(dataset, mode="fit")
+        inputs = return_single_batch(
+            dataset,
+            mode="fit",
+        )  # split_file="modelforge/tests/qm9tut/split.npz")
         output = initialized_model(inputs)
         print(output)
-        assert output.shape[0] == 64
-        assert output.shape[1] == 1
+        if isinstance(output, dict):
+            assert output["scalar_representation"].shape[0] == 1088
+        else:
+            assert output.shape[0] == 64
+            assert output.shape[1] == 1
+
+
+# @pytest.mark.parametrize("dataset", DATASETS)
+# def test_forward_pass_schnetpack_painn(dataset):
+#     initialized_model = setup_painn()
+#     inputs = return_single_batch(
+#         dataset,
+#         mode="fit",
+#     )  # split_file="modelforge/tests/qm9tut/split.npz")
+#     output = initialized_model(inputs)
+#     print(output)
+#     assert output.shape[0] == 64
+#     assert output.shape[1] == 1
 
 
 def test_pairlist_simple_data():
```

Comment:
If my comment is accurate about the pair list computation being off, these tests need to be rewritten because they aren't catching the error I saw.